.gitignore

```diff
@@ -68,8 +68,9 @@ typings/
 # Yarn Integrity file
 .yarn-integrity
 
-# dotenv environment variables file
+# environment variables file
 .env
+.envrc
```

Comment:
Since you are adding this file into the repo it should not be part of the `.gitignore`. We should add it to the `.npmignore` file instead.

Reply:
ok, it makes sense so we don't have it published 👌